featuretools/primitives/standard/transform_primitive.py

```diff
@@ -212,6 +214,7 @@ class Week(TransformPrimitive):
 
     def get_function(self):
         def week(vals):
+            warnings.filterwarnings("ignore", message="Series.dt.weekofyear and Series.dt.week have been deprecated.", module="featuretools")
```

Comment:
this line is a little long (142), can we reduce line length

Reply:
:+1: